lib/msal-react/src/MsalContext.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import { IPublicClientApplication, AccountInfo } from "@azure/msal-browser";
+import { PublicClientApplication } from "../../msal-browser/dist/src";
```

Comment:
This should come from the `@azure/msal-browser` import.

Reply:
Thanks for catching this!